gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
 assertk = "0.26-SNAPSHOT"
-kotlin = "1.5.30"
+kotlin = "1.8.10"
 [libraries]
 kotlin-gradle = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
-dokka-gradle = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.5.0" }
+dokka-gradle = { module = "org.jetbrains.dokka:dokka-gradle-plugin", version = "1.8.10" }
 detekt-gradle = { module = "io.gitlab.arturbosch.detekt:detekt-gradle-plugin", version = "1.19.0" }
-kotlin-coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.5.2" }
-opentest4k = { module = "com.willowtreeapps.opentest4k:opentest4k", version = "1.2.2" }
+kotlin-coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.7.0-Beta" }
```

Comment:
This beta is the first with all the targets needed. I would wait to release a new version until 1.7.0 final is out, unless you also want to do a beta.

Reply:
Yeah because of the low frequency of releases think it's best to wait